### DISCUSSION TITLE
Abaqus input files to OpenRadioss

### DISCUSSION THREAD

**Discussion by Bidhendi** [OpenRadioss/OpenRadioss#1084] [Q&A]
Hi All,

I use Abaqus for pre-post and for analysis. I have not used OpenRad before but I am interested in trying it as a solver. Do you think there is a possibility that one could perform the preprocessing (geometry, meshing, material, and BC) in Abaqus and then translate the .inp to a Rad input file? What is actually the Rad input file format? Any feedback or examples are much appreciated.

Thank you!

**Comment by PaulAltair** [comment]
Hi, thank you for your interest, there are commercial packages (e.g. Altair HyperWorks), that may 'convert' Abaqus .inp input files to Radioss input format, but the conversion may not be '1 to 1' or complete, depending on features used. 

Regarding the .rad (Radioss 'Block Format' input)
The OpenRadioss/Altair Radioss native input format is fully documented: [Confluence OpenRadioss Documentation](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/4816906/OpenRadioss+User+Documentation)

There is also a collection of example models here that you can look at for reference: [OpenRadioss Example Models](https://openradioss.atlassian.net/wiki/spaces/OPENRADIOSS/pages/8749071/Example+Models)

OpenRadioss may also use LS-DYNA keyword input format (also documented in the Radioss documentation at the first link) if you have any familiarity with that? So an Alternative approach may be to convert your Abaqus input to LS-DYNA format?

**Reply by Bidhendi** [reply]
Hi Paul, Thanks so much for your reply. Good points to start with.
